### ISSUE TITLE
I have fixed the problem of WebSocketConnection: Connection is already been closed

### ISSUE THREAD

**Issue by ssqwerljack** [AlizerUncaged/desktop-waifu#67]

**Comment by DarkUrine** [comment]
Thanks for the first fix! But yeah I am also having the same issue you are now haha. Hopefully someone will respond with a solution sooner than later.

**Comment by LunasApocalypse** [comment]
How about the voice? Mine only reply one time with no voice and can't do anything after that.

**Comment by chancla2** [comment]
lol same here, i try to put a wait between message and message, but just a few questions later and the error pop.

**Comment by WalelaRose** [comment]
Yeah, it can connect now but it doesn't hear me, it just outputs gibberish from me

**Comment by Spy4070D** [comment]
I believe this error is likely related to the limited free messages for guest users on Character.AI.
